app/routes/admin/scene.py

```diff
@@ -118,3 +119,27 @@ async def get(self):
 
         except Exception as e:
             self.write_error(status_code=500, message=str(e))
+
+
+class AdminSceneScreenshotHandler(BaseAdminAPIHandler):
```

Comment:
im unaware of some context.. this one is for manually invoking a screenshot and update?

Reply:
This one is for updating the screenshot URL link, so we don't have to get entire scene JSON and send updated version for updating just 1 field (screenshot_url)